### DISCUSSION TITLE
*fixed* 'Not an editor command: Neorg' and no styling/conceal

### DISCUSSION THREAD

**Discussion by mbpowers** [nvim-neorg/neorg#1118] [Q&A] [answered]
Below is what I was gonna post but I figured it out, I think my Packer is just old and messed up and probably unique to my config, but I already wrote this so. 

The fix was I had to to remove the `function()` and `end` from the `config = ...` (relevant info https://github.com/wbthomason/packer.nvim/issues/908). Here is what worked for me:

```
return require('packer').startup(function()
    use { 'wbthomason/packer.nvim' }

    ...other plugins...

    use { "nvim-neorg/neorg",
        config = require('neorg').setup {
            load = {
                ["core.defaults"] = {}, -- Loads default behaviour
                ["core.concealer"] = {}, -- Adds pretty icons to your documents
                ["core.dirman"] = { -- Manages Neorg workspaces
                    config = {
                        workspaces = {
                            notes = "~/notes",
                        },
                    },
                },
            },
        },
        run = ":Neorg sync-parsers",
        requires = "nvim-lua/plenary.nvim",
    }
end)
```

---
### Original post

Hey guys not sure what I'm doing wrong, as the title says I have no `:Neorg` commands TS parser doesn't automatically install and no styling/conceal. I do however have `:h neorg` and the like. I assume this is just an error on my end.

I have tried manually running `TSInstall norg` and have `conceallevel=2` and surrounding symbols are hidden but no text styles or icons are set (`*bold*` shows `bold` but isn't actually bold and `***` isn't replaced). Before `TSInstall norg` no conceal/syntax highlighting at all.

I have set my font to MesloLGM Nerd Font (side question, is this the only nerd-font that works, or can I use JetBrains?).

Here is some info on my setup:
I am on the current nightly-bin from the AUR
```
$ nvim --version
NVIM v0.10.0-dev-835+gdd09630a2
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

relevant part of my plugins.lua using packer (straight from README)
```
return require('packer').startup(function()
    use { 'wbthomason/packer.nvim' }

    ...other plugins...

    use { "nvim-neorg/neorg",
        config = function()
            require('neorg').setup {
                load = {
                    ["core.defaults"] = {}, -- Loads default behaviour
                    ["core.concealer"] = {}, -- Adds pretty icons to your documents
                    ["core.dirman"] = { -- Manages Neorg workspaces
                        config = {
                            workspaces = {
                                notes = "~/notes",
                            },
                        },
                    },
                },
            }
        end,
        run = ":Neorg sync-parsers",
        requires = "nvim-lua/plenary.nvim",
    }
end)
```

**Comment by max397574** [comment]
packer being weird
I can't see why the old thing wouldn't work. Back then when I used packer it always worked like this

I'm confused
is the first code block the working or the broken one?

**Reply by mbpowers** [reply]
The first one is the working one, the one below is the broken one. The weird thing is in the issue I linked, that person fixed their issue by doing the inverse of what fixed mine :shrug:. It has been a year or two since I made this config though so it may be time to spring clean.

**Comment by mbpowers** [comment, marked as answer]
The answer is in the post.
